### PR TITLE
TAC模式读已提交的隔离级别bug修复

### PR DESCRIPTION
**Pull request by zkyoma**
Make sure that:

- [x] You have read the [contribution guidelines](https://dromara.org/projects/hmily/contributor/).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

for issue #363

### CODE REVIEW

File: hmily-demo/hmily-demo-tac/hmily-demo-tac-dubbo/hmily-demo-tac-dubbo-order/src/main/java/org/dromara/hmily/demo/tac/dubbo/order/service/PaymentService.java [hunk 1]
```diff
@@ -97,6 +97,7 @@ public interface PaymentService {
      * 订单支付.
      *
      * @param order 订单实体
+     * @param type 类型
      */
-    String makePaymentWithReadCommitted(Order order);
+    String makePaymentWithReadCommitted(Order order, int type);
```
Comment (cherrylzhao) [inline hmily-demo/hmily-demo-tac/hmily-demo-tac-dubbo/hmily-demo-tac-dubbo-order/src/main/java/org/dromara/hmily/demo/tac/dubbo/order/service/PaymentService.java:102]: type 的含义不明确，代码中避免直接用1，2进行区分，建议使用enum

Comment (zkyoma) [reply hmily-demo/hmily-demo-tac/hmily-demo-tac-dubbo/hmily-demo-tac-dubbo-order/src/main/java/org/dromara/hmily/demo/tac/dubbo/order/service/PaymentService.java:102]: 好的

File: hmily-demo/hmily-demo-tac/hmily-demo-tac-dubbo/hmily-demo-tac-dubbo-order/src/main/java/org/dromara/hmily/demo/tac/dubbo/order/enums/ReadCommittedTransactionEnum.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+/**
+ * The enum transaction enum.
+ *
+ * @author zhangzhi
+ */
+@RequiredArgsConstructor
+@Getter
+public enum ReadCommittedTransactionEnum {
+
+    READ_COMMITTED_TRANSACTION_ALL(1, "读已提交隔离级别的事务, 包括更新、查询操作"),
```
Comment (cherrylzhao) [inline hmily-demo/hmily-demo-tac/hmily-demo-tac-dubbo/hmily-demo-tac-dubbo-order/src/main/java/org/dromara/hmily/demo/tac/dubbo/order/enums/ReadCommittedTransactionEnum.java:15]: 建议替换为READ_WRITE, READ_ONLY 两个枚举值

Comment (cherrylzhao) [inline hmily-demo/hmily-demo-tac/hmily-demo-tac-dubbo/hmily-demo-tac-dubbo-order/src/main/java/org/dromara/hmily/demo/tac/dubbo/order/enums/ReadCommittedTransactionEnum.java:15]: 建议替换为TRANSACTION_READ_WRITE

File: hmily-demo/hmily-demo-tac/hmily-demo-tac-dubbo/hmily-demo-tac-dubbo-order/src/main/java/org/dromara/hmily/demo/tac/dubbo/order/enums/ReadCommittedTransactionEnum.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+    READ_COMMITTED_TRANSACTION_JUST_SELECT(2, "读已提交隔离级别的事务, 只有查询操作");
```
Comment (cherrylzhao) [inline hmily-demo/hmily-demo-tac/hmily-demo-tac-dubbo/hmily-demo-tac-dubbo-order/src/main/java/org/dromara/hmily/demo/tac/dubbo/order/enums/ReadCommittedTransactionEnum.java:17]: 建议替换为TRANSACTION_READ_ONLY

File: hmily-demo/hmily-demo-tac/hmily-demo-tac-dubbo/hmily-demo-tac-dubbo-order/src/main/java/org/dromara/hmily/demo/tac/dubbo/order/enums/ReadCommittedTransactionEnum.java (hunk 2 above)
Comment (zkyoma) [reply hmily-demo/hmily-demo-tac/hmily-demo-tac-dubbo/hmily-demo-tac-dubbo-order/src/main/java/org/dromara/hmily/demo/tac/dubbo/order/enums/ReadCommittedTransactionEnum.java]: 已修改

File: hmily-demo/hmily-demo-tac/hmily-demo-tac-dubbo/hmily-demo-tac-dubbo-order/src/main/java/org/dromara/hmily/demo/tac/dubbo/order/enums/ReadCommittedTransactionEnum.java (hunk 3 above)
Comment (zkyoma) [reply hmily-demo/hmily-demo-tac/hmily-demo-tac-dubbo/hmily-demo-tac-dubbo-order/src/main/java/org/dromara/hmily/demo/tac/dubbo/order/enums/ReadCommittedTransactionEnum.java]: 已修改

**Pull request merged**